hack/ci/run-etcd-launcher-tests.sh

```diff
@@ -34,18 +34,18 @@ export KIND_CLUSTER_NAME="${SEED_NAME:-kubermatic}"
 source hack/ci/setup-kind-cluster.sh
 source hack/ci/setup-kubermatic-backups-in-kind.sh
 
-echodate "Creating UI DigitalOcean preset..."
-cat << EOF > preset-digitalocean.yaml
+echodate "Creating Hetzner preset..."
+cat << EOF > preset-hetzner.yaml
 apiVersion: kubermatic.k8c.io/v1
 kind: Preset
 metadata:
-  name: e2e-digitalocean
+  name: e2e-hetzner
   namespace: kubermatic
 spec:
-  digitalocean:
-    token: ${DO_E2E_TESTS_TOKEN}
+  hetzner:
+    token: ${HZ_E2E_TOKEN}
 EOF
-retry 2 kubectl apply -f preset-digitalocean.yaml
+retry 2 kubectl apply -f preset-hetzner.yaml
```

Comment:
Why don't we use the `kubeadm` ("BYOC") provider for this (and perhaps other tests that only concern the control plane)? That makes sure we do not rely on a cloud API to be up and running when not needed, as we only get the control plane. But that would suffice for `etcd-launcher` tests.

Reply:
Yeah, I get that, I don't mean to suggest that we completely drop using a cloud provider like Hetzner, but how do you think about using `byoc` where it makes sense (like the etcd-launcher test above)?

Reply:
We only have one single function to create a cluster in our tests, and that one uses Hetzner. I did not want to duplicate code needlessly.

Reply:
Okay, sounds reasonable for now.